File: backend/src/main/java/com/lgdlong/backend/dto/AdminUserUpdate.java
package com.lgdlong.backend.dto;

import com.lgdlong.backend.enums.*;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AdminUserUpdate {
    private String username;
    private String displayName;
    // Email cannot be changed by admin
    private String email;
    // Phone cannot be changed by admin
    private String phone;
    private String picUrl; // avatarUrl
    private UserStatus status;
}

File: frontend/src/interfaces/AdminUserUpdate.ts
import { UserStatus } from "../enums/UserEnums";

export interface AdminUserUpdate {
  username: string;
  displayName: string;
  email?: string; // ← CANT CHANGE
  phone?: string; // ← CANT CHANGE
  picUrl?: string; // ← made optional
  status?: UserStatus; // ← made optional
}
